src/services/notifications.service.ts

```diff
@@ -3,13 +3,15 @@ import * as signalRMsgPack from '@microsoft/signalr-protocol-msgpack';
 
 import { NotificationType } from '../enums/notificationType';
 
+import { LogService } from '../abstractions'
 import { ApiService } from '../abstractions/api.service';
 import { AppIdService } from '../abstractions/appId.service';
 import { EnvironmentService } from '../abstractions/environment.service';
 import { NotificationsService as NotificationsServiceAbstraction } from '../abstractions/notifications.service';
 import { SyncService } from '../abstractions/sync.service';
 import { UserService } from '../abstractions/user.service';
 import { VaultTimeoutService } from '../abstractions/vaultTimeout.service';
+import { ConsoleLogService } from '../cli/services/consoleLog.service';
```

Comment:
We should not be mixing in CLI services here. Dependencies need to be passed in.

Reply:
How do you feel about a base ConsoleLogService, which the CLI one will inherit from?

Reply:
That would be fine